engine/swarm/networking.md

```diff
@@ -233,13 +233,16 @@ from any swarm node which is joined to the swarm and is in a `running` state.
 service's external clients to an individual swarm node, without the client
 needing to know how many nodes are participating in the service or their
 IP addresses or ports. You don't need to publish ports which are used between
-services on the same network. For instance, if you have a WordPress service
-which stores its data in a MySQL service, and they are connected to the same
-overlay network, you do not need to publish the MySQL port to the client, only
-the WordPress HTTP port.
-
-Service discovery can work in two different ways, using a virtual IP (VIP) or
-DNS round robin (DNSRR). You can configure this per service.
+services on the same network. For instance, if you have a [WordPress service
```

Comment:
This is personal preference, but I'd wrap after `if you have a` and then put the whole link text and link all on one line, so your next newline would be after the `,`. It breaks wrapping  conventions but it makes you less likely to accidentally break your link by having an inadvertent space or something at the wrong place.